CliClient/locales/fr_FR.po

```diff
@@ -643,16 +645,18 @@ msgid "Cannot initialize synchroniser."
 msgstr "Impossible d'initialiser la synchronisation."
 
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:179
+#, fuzzy
 msgid "Starting synchronisation..."
-msgstr "Commencement de la synchronisation..."
+msgstr "Lancement de la synchronisation..."
 
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:202
 msgid "Downloading resources..."
 msgstr "Téléchargement des ressources..."
 
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:226
+#, fuzzy
 msgid "Cancelling... Please wait."
-msgstr "Annulation... Veuillez attendre."
+msgstr "Annulation... Veuillez patienter."
```

Comment:
I don't understand. You've changed the translation, but somehow you've also added a fuzzy flag? So is it a fuzzy translation or not? I don't think it is fuzzy as your translation is good.

In practice what will happen is that this text will now appear in English in the app because fuzzy translations are ignored.

Reply:
Sorry, I didn't realise it would have this effect. I'll get theses fixes in. I'll create another pull request, just realised I forogt to create a separate branch for this.